patches/chrome-browser-ui-views-tabs-tab.cc.patch

```diff
@@ -10,7 +10,15 @@ index 7b4670597248635874b2b1650b1eae40fddb52e7..1888150f0ad27c60b19bc451c6a70eb7
      } else if (showing_close_button_) {
        // Allow the title to overlay the close button's empty border padding.
        title_right = close_x - after_title_padding;
-@@ -1009,6 +1010,7 @@ void Tab::UpdateIconVisibility() {
+@@ -643,6 +644,7 @@ void Tab::OnGestureEvent(ui::GestureEvent* event) {
+ }
+ 
+ std::u16string Tab::GetTooltipText(const gfx::Point& p) const {
++  BRAVE_TAB_HOVER_TOOLTIPS
```

Comment:
It could be, but I think updating [TabContainer::AddTab](https://source.chromium.org/chromium/chromium/src/+/main:chrome/browser/ui/views/tabs/tab_container.cc;l=219;bpv=1;bpt=1?q=TabContainer::AddTab) to create a BraveTab instead of a ChromeTab looks like it would probably require a patch too, and might be a bit more complex. I could be wrong about that though, I'm pretty new to the Brave way of patching :laughing: 

I also looked into patching `GetTooltipText` directly, like the other things in the `chromium_src` `tab` but `tab.h` has a `static` and `non-static` `GetTooltipText` defined, which do slightly different things.

I'm happy to try and setup a `BraveTab` class, but it seems like a slightly bigger refactor, and is maybe worth it's own ticket and moving a chunk of the existing stuff into it.

Reply:
there are examples where we rename the original class and then subclass our version as the original name